website/docs/cli.md

```diff
@@ -114,6 +114,12 @@ Running the command will copy the relevant theme files to your site folder. You
 | `--danger`         | Allow swizzling of unstable components |
 | `--typescript`     | Swizzle TypeScript components          |
 
+An example to use `--danger` flag let's consider the below code:
```

Comment:
The wording feels a bit weird to me, can you find a better sentence? Also worth explaining what we mean by "unstable components" (=> components that have a higher risk of breaking changes due to internal refactorings).


You could put all this in an admonition maybe?